libcloudforensics/providers/gcp/forensics.py

```diff
@@ -88,28 +90,23 @@ def CreateDiskCopy(
     snapshot.Delete()
     logger.debug('Snapshot {0:s} deleted.'.format(snapshot.name))
 
-  except RefreshError as exception:
-    error_msg = ('Something is wrong with your gcloud access token: '
-                 '{0:s}.').format(exception)
-    raise RuntimeError(error_msg)
-  except DefaultCredentialsError as exception:
-    error_msg = (
-        'Something is wrong with your Application Default '
-        'Credentials. '
-        'Try running:\n  $ gcloud auth application-default login')
-    raise RuntimeError(error_msg)
+  except (RefreshError, DefaultCredentialsError) as exception:
+    raise errors.CredentialsConfigurationError(
+        'Something is wrong with your Application Default Credentials. Try '
+        'running: $ gcloud auth application-default login: {0!s}'.format(
+            exception), __name__)
   except HttpError as exception:
     if exception.resp.status == 403:
       raise RuntimeError(
```

Comment:
Would this be  a CredentialsConfigurationError ?

Reply:
Changed